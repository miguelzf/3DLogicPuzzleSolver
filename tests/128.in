- - - - - -
- - - L - -
- - - - - -
- - - - - Y
- - - - - G
- - - - - P

- - - - - -
- - - - - R
- - - - - L
- - - - - -
- - - - - -
- - - Y - -

- - - - - -
- - - - - R
- - - - - B
B - - - G -
P - - - - -
- - - - - -
