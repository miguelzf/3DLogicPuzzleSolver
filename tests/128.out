
Solution:
r  r  r  r  r  r  
r  -  -  L  l  l  
r  r  r  r  r  r  
y  y  y  y  y  Y  
y  g  g  g  g  G  
y  g  p  p  p  P  
================
y  g  p  -  -  r  ||  r  r  r  r  l  r  
y  g  p  -  -  R  ||  l  l  l  l  l  R  
y  g  p  -  -  L  ||  l  b  b  b  b  B  
y  g  p  p  p  p  ||  B  b  -  -  G  g  
y  g  g  g  g  p  ||  P  -  -  -  -  g  
y  y  y  Y  g  g  ||  g  g  g  g  g  g  
